### PR TITLE
Reexport Web3 objects after module augmentation and update README.md with example

### PR DESCRIPTION
**Pull request by Muhammad-Altabba**
Implements https://github.com/web3/web3.js/issues/5596 and https://github.com/web3/web3.js/issues/5569

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,27 @@ npm install @chainsafe/web3.js-chainlink-plugin
 yarn add @chainsafe/web3.js-chainlink-plugin
 ```
 
+## Using this plugin
+
+You can use this plugin to get the price of token-pairs from a Chainlink smart contract.
+
+And here is a code snippet that get the Price of the pair `Link/Eth` from the Ethereum mainnet.
+However, if you are connected to another ethereum network other than the mainnet, or you would like to get the price of another pair, replace `MainnetPriceFeeds.LinkEth`, in the code bellow, with the address of the intended Chainlink smart contract that is deployed on the network that you are connected to. And you may check https://docs.chain.link/docs/data-feeds/price-feeds/addresses/ for the most updated mainnet and testnet addresses.
```
Comment (luu-alex) [inline README.md:35]:
```suggestion
However, if you are connected to another Ethereum network other than the mainnet, or you would like to get the price of another pair, replace `MainnetPriceFeeds.LinkEth`, in the code below, with the address of the intended Chainlink smart contract that is deployed on the network that you are connected to. And you may check https://docs.chain.link/docs/data-feeds/price-feeds/addresses/ for the most updated mainnet and testnet addresses.
```

**Review by Fqc301033 — APPROVED** [review]

**Review by nikoulai — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -27,6 +27,27 @@ npm install @chainsafe/web3.js-chainlink-plugin
...
+
+And here is a code snippet that get the Price of the pair `Link/Eth` from the Ethereum mainnet.
+However, if you are connected to another Ethereum network other than the mainnet, or you would like to get the price of another pair, replace `MainnetPriceFeeds.LinkEth`, in the code below, with the address of the intended Chainlink smart contract that is deployed on the network that you are connected to. And you may check https://docs.chain.link/docs/data-feeds/price-feeds/addresses/ for the most updated mainnet and testnet addresses.
+
+```typescript
+import { ChainlinkPlugin, MainnetPriceFeeds } from '@chainsafe/web3.js-chainlink-plugin';
+
+import Web3 from 'web3';
+
+const web3 = new Web3('YOUR_ETHEREUM_NODE'); // Use a node connectd to mainnet, if you will keep using `MainnetPriceFeeds.LinkEth` bellow.
```
Comment (jdevcs) [inline README.md:42]:
```suggestion
const web3 = new Web3('YOUR_ETHEREUM_NODE'); // Use a node connected to mainnet, if you will keep using `MainnetPriceFeeds.LinkEth` below.
```

File: README.md [hunk 3]
```diff
@@ -27,6 +27,27 @@ npm install @chainsafe/web3.js-chainlink-plugin
 yarn add @chainsafe/web3.js-chainlink-plugin
 ```
 
+## Using this plugin
+
+You can use this plugin to get the price of token-pairs from a Chainlink smart contract.
+
+And here is a code snippet that get the Price of the pair `Link/Eth` from the Ethereum mainnet.
+However, if you are connected to another Ethereum network other than the mainnet, or you would like to get the price of another pair, replace `MainnetPriceFeeds.LinkEth`, in the code below, with the address of the intended Chainlink smart contract that is deployed on the network that you are connected to. And you may check https://docs.chain.link/docs/data-feeds/price-feeds/addresses/ for the most updated mainnet and testnet addresses.
```
Comment (jdevcs) [inline README.md:35]:
```suggestion
However, if you are connected to another Ethereum network other than the mainnet, or you would like to get the price of another pair, replace `MainnetPriceFeeds.LinkEth`, in the code below, with the address of the intended Chainlink smart contract that is deployed on the network that you are connected to, and you may check https://docs.chain.link/docs/data-feeds/price-feeds/addresses/ for the most updated mainnet and testnet addresses.
```

File: README.md [hunk 4]
```diff
@@ -27,6 +27,27 @@ npm install @chainsafe/web3.js-chainlink-plugin
 yarn add @chainsafe/web3.js-chainlink-plugin
 ```
 
+## Using this plugin
+
+You can use this plugin to get the price of token-pairs from a Chainlink smart contract.
+
+And here is a code snippet that get the Price of the pair `Link/Eth` from the Ethereum mainnet.
```
Comment (jdevcs) [inline README.md:34]:
```suggestion
Following is a code snippet is for getting Price of the pair `Link/Eth` from the Ethereum mainnet.
```

**Comment by jdevcs** [comment]
LGTM, except few changes. Thanks @Muhammad-Altabba

**Review by luu-alex — APPROVED** [review]

**Pull request merged**